modules/installation/pages/install-intro.adoc

```diff
@@ -11,24 +11,11 @@ include::_partials/entities.adoc[]
 endif::[]
 
 
+{productname} provides absolute control over your Linux environment.
+System administrators often have tens, hundreds, or even thousands of client machines they need to look after, all of which require ongoing maintenance, updates, and lifecycle management.
+{productname} allows you to manage all your Linux clients in one place, with one easy-to-understand dashboard.
 
-{productname} is a solution for organizations that require absolute control over maintenance and package deployment on their servers.
-It lets you manage large sets of Linux systems and keep them up to date, with automated software management, asset management, and system provisioning.
-{productname} allows you to maintain a high level of security while effectively managing system life-cycle requirements.
-
-{productname} uses _Salt_ to provide event-driven configuration and management control.
-The Salt-master orchestrates thousands of Salt-minions ({productname} Clients) using remote execution.
+{productname} can be integrated with your network infrastructure in multiple ways, and most tasks can be automated, with reports issued so you always know the status of your client machines.
+Because {productname} allows you to manage large numbers of systems and automatically keep them up to date, it helps to improve overall security, and provides extensive asset management and provisioning capability.
 
 {productname} is fully compatible with Red Hat Satellite Server and offers seamless management of both {sle} and {rhel} client systems.
```

Comment:
@joesusecom should we remove the term 'fully compatible with RedHat satelliete'. Yesterday I learned that Satellite 5 is now not maintained anymore by RedHat. I think Satellite would now mean version 6 which is not true. Or can we change the sentence to make it more clear what we mean?

Reply:
I wondered about the "fully compatible" language, it seems like it could become problematic.